docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -1893,6 +1893,64 @@ User.order(:foo).each
 
 * https://rails.rubystyle.guide#find-each
 
+== Rails/ForeignKeyName
```

Comment:
Same here for docs - they will be automatically generated later.

Reply:
Nice, this has been changed since I actively contributed a year ago 👍